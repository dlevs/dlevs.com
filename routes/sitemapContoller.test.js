'use strict';

const { testControllerSnapshots, getMockControllerParams } = require('../tests/testLib/testUtils');
const sitemapController = require('./sitemapController')(getMockControllerParams());

testControllerSnapshots(sitemapController.controller, {
	html: { contexts: [{}] },
	xml: { contexts: [{}] },
});
