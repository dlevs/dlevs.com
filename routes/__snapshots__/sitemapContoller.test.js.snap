// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`html data passed to template with ctx {} 1`] = `
Object {
  "ctx": Object {
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
  },
  "renderedData": Object {
    "breadcrumb": Breadcrumb {
      "parts": Array [
        Object {
          "name": "Home",
          "path": "/",
          "slug": "",
        },
        Object {
          "name": "Sitemap",
          "path": "/sitemap",
          "slug": "sitemap",
        },
      ],
    },
    "meta": Object {
      "title": "Sitemap",
    },
    "origin": undefined,
    "pages": Array [
      Object {
        "name": "Home",
        "path": "/",
      },
      Object {
        "name": "Travel",
        "path": "/travel",
        "posts": Array [
          Object {
            "name": "Japan",
            "path": "/travel/japan",
          },
        ],
      },
    ],
  },
}
`;

exports[`html data passed to template with ctx {} 2`] = `
Object {
  "ctx": Object {
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
  },
  "renderedData": Object {
    "breadcrumb": Breadcrumb {
      "parts": Array [
        Object {
          "name": "Home",
          "path": "/",
          "slug": "",
        },
        Object {
          "name": "Sitemap",
          "path": "/sitemap",
          "slug": "sitemap",
        },
      ],
    },
    "meta": Object {
      "title": "Sitemap",
    },
    "origin": undefined,
    "pages": Array [],
  },
}
`;

exports[`xml data passed to template with ctx {} 1`] = `
Object {
  "ctx": Object {
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
    "type": "xml",
  },
  "renderedData": Object {
    "breadcrumb": Breadcrumb {
      "parts": Array [
        Object {
          "name": "Home",
          "path": "/",
          "slug": "",
        },
        Object {
          "name": "Sitemap",
          "path": "/sitemap",
          "slug": "sitemap",
        },
      ],
    },
    "meta": Object {
      "title": "Sitemap",
    },
    "origin": undefined,
    "pages": Array [
      Object {
        "name": "Home",
        "path": "/",
      },
      Object {
        "name": "Travel",
        "path": "/travel",
        "posts": Array [
          Object {
            "name": "Japan",
            "path": "/travel/japan",
          },
        ],
      },
    ],
  },
}
`;

exports[`xml data passed to template with ctx {} 2`] = `
Object {
  "ctx": Object {
    "query": Object {},
    "render": [Function],
    "state": Object {
      "CANONICAL_URL": "DUMMY_CANONICAL_URL",
    },
    "type": "xml",
  },
  "renderedData": Object {
    "breadcrumb": Breadcrumb {
      "parts": Array [
        Object {
          "name": "Home",
          "path": "/",
          "slug": "",
        },
        Object {
          "name": "Sitemap",
          "path": "/sitemap",
          "slug": "sitemap",
        },
      ],
    },
    "meta": Object {
      "title": "Sitemap",
    },
    "origin": undefined,
    "pages": Array [],
  },
}
`;
